### PR TITLE
Adds a rules table row for bazel-bats.

### LINKED ISSUE #70: FYI: bazel build rules for bats (bash automated testing system)
Rules at:
https://github.com/filmil/bazel-bats

### PR DESCRIPTION
**Pull request by filmil**
This is a set of bazel rules for bats-core (https://github.com/bats-core/bats-core).

Fixes #70.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,14 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
         </ul>
       </td>
     </tr>
+    <tr>
+      <td>bats (Bash automated testing system)</td>
+      <td>
+        <ul>
+          <li><a href="https://github.com/filmil/bazel-bats">filmil/bazel-bats</a></li>
```
Comment (jin) [inline README.md:101]: Could you add this into the Shell row instead?

Comment (filmil) [reply README.md:101]: Will do when able.

Comment (filmil) [reply README.md]: Done.

**Comment by jin** [comment]
Thanks!

**Pull request merged**